Assets/MixedRealityToolkit.Staging/UnityAR/Editor/ConfigurationChecker/Microsoft.MixedReality.UnityAR.ConfigurationChecker.asmdef

```diff
@@ -0,0 +1,16 @@
+{
+    "name": "Microsoft.MixedReality.UnityAR.ConfigCheck.asmdef",
```

Comment:
Can we re-use the `Microsoft.MixedReality.Providers.UnityAR.Editor` assembly for this?

Reply:
I was attempting to eliminate reference issues if the UnityAR provider assembly is not buildable a the time this code needs to run.

Thoughts?

Reply:
Ah okay, makes sense

Reply:
confirmed that merging this into the editor assembly will break on platforms that rely on the runtime assembly to successfully compile. i recommend we keep the separate config checker assembly